tests/test_keras_integration.py

```diff
@@ -140,6 +146,13 @@ def fin():
 
 
 def test_show_prediction_nodeps(show_nodeps, keras_clf, cat_dog_image):
-    with pytest.warns(UserWarning):
-        expl = show_nodeps(keras_clf, cat_dog_image)
-    assert isinstance(expl, Explanation)
\ No newline at end of file
+    doc, image = cat_dog_image
+    with pytest.warns(UserWarning) as rec:
+        expl = show_nodeps(keras_clf, doc)
+    assert 'dependencies' in str(rec[-1].message)
+    assert isinstance(expl, Explanation)
+
+
+def test_explain_prediction_not_supported():
+    res = eli5.explain_prediction(Sequential(), np.zeros((0,)))
```

Comment:
is this check faling only because of doc shape, or because of model as well? If that's only doc, what do you think about adding a case where it fails because of a model?

Reply:
Exactly, here it fails because of `doc` and short-circuits. Added another test case.